providers/openebs/cspc_csi_upgrade/README.md

```diff
@@ -0,0 +1,27 @@
+## Test Case to upgrade the cspc csi components
+
+## Description
+   - This TestCase is capable of Upgrade the OpenEBS CSPC CSI related components.
```

Comment:
This test case is capable of upgrading the OpenEBS CSPC pool and cStor-csi volume components